docs/output.md

```diff
@@ -93,7 +99,18 @@ Output Summaries:
 
 ## Tool details
 
-### Annotation tools
+### Taxonomic classification tool
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `taxonomic_classification/mmseqs2_createtsv/`
+  - `<samplename>/`:
+    - `*.tsv`: tab seperated table containing the taxonomic lineage of every contig when available.
+- `reports/<workflow>/<workflow>_complete_summary_taxonomy.tsv.gz`: tab seperated table containing the concatenated results from the <workflow> summary tables along with the taxonomic classification if the parameter `run_taxonomic_classification` is called.
+</details>
+
+[MMseqs2](https://github.com/soedinglab/MMseqs2) classifies the taxonomic lineage of contigs based on the last common ancestor. The inferred taxonomic lineages are included in the final workflow summaries to annotate the potential source bacteria of the identified AMPs, ARGs, and/or BGCs.
 
 [Pyrodigal](#pyrodigal), [Prodigal](#prodigal), [Prokka](#prokka), [Bakta](#bakta)
```

Comment:
What is this line meant to be doing?

Reply:
i removed the heading by mistake .. oopsy